rhc-ose-ansible/ose-provision.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
@sabre1041 shouldn't this be in a playbook, instead of in the root of the project?